README.md

```diff
@@ -20,19 +22,60 @@
 
 - ### Install Docker CLI in Ubuntu.
 
-  - #### Follow Steps on: [https://docs.docker.com/engine/install/ubuntu/](https://docs.docker.com/engine/install/ubuntu/)
+  - #### Follow steps on: [https://docs.docker.com/engine/install/ubuntu/](https://docs.docker.com/engine/install/ubuntu/)
   - #### Run the command: `dockerd` to start the docker daemon
 
 - ### Open ubuntu terminal and clone repository.
 
-- ### Navigate to cloned repo and enter `code .` to open the repo in vsCode using remote connection.
+- ### Navigate to cloned repo and enter `code .` to open the repo in VS Code using remote connection.
+
+- ### Remote into Dev Container <small>(using green box in the bottom left)</small>
+
+## VERY IMPORTANT!!!
+
+### We did not get to the point of setting up dev certificates. This causes the issue of not allowing the Fetch API to access endpoints
+
+- ### To fix this, you can create a file called .env within the client directory and paste `export NODE_TLS_REJECT_UNAUTHORIZED=0` into it
+
+# Development
+
+## How to Start Development Server
+
+- ### Open the Project in the Dev Container
+
+- ### From the Root Project's Root Directory run the commands: `cd server` then `dotnet run`
+
+  - #### This will start up the backend of the program

```

Comment:
this is put as a bullet which represents a step. we should probably have it with the line above